TROUBLESHOOTING.md

```diff
@@ -39,6 +39,38 @@ be taken as relative to the workspace root.
 
 Note that if you are coming to Pylance from using the Microsoft Python Language Server, this setting has changed from `python.autoComplete.extraPaths` to `python.analysis.extraPaths`.
 
+### Editable install modules not found
+
+[PEP 660](https://peps.python.org/pep-0660/) enables build backends (ex. setuptools) to
+use import hooks to direct the [import machinery](https://docs.python.org/3/reference/import.html)
+to the package's source files rather than using a `.pth` file. Import hooks can provide
+an editable installation that is a more accurate representation of your real installation.
+However, because resolving module locations using an import hook requires executing Python
+code, they are not usable by Pylance, Pyright, and other static analysis tools. Therefore,
+if your editable install is configured to use import hooks, Pylance/Pyright will be unable
+to find the corresponding source files.
+
+If you want to use static analysis tools and you are willing to accept a lower-fidelity
```

Comment:
I recommend deleting "and you are willing to accept a lower-fidelity editable install". There really isn't a choice here. If you want to use static analysis tools, you will need to use ".pth" files. (I'm also skeptical of the assertion that the fidelity difference is meaningful to all but a tiny percentage of users who perform editable installs, so I don't think we should emphasize this point.)